1.1/spec/10-Part-07.adoc

```diff
@@ -1,94 +1,100 @@
-=== Topology Vocabulary Extension (relation_family)
+== Topology Vocabulary Extension (relation_family)
 
 This clause establishes the _Topology Vocabulary Extension (relation_family)_ parameterized requirements class, with IRI `/req/topology-vocab-extension`, which has a single corresponding conformance class _Topology Vocabulary Extension (relation_family)_, with IRI `/conf/topology-vocab-extension`. This requirements class defines a vocabulary for asserting and querying topological relations between spatial objects. The class is parameterized so that different families of topological relations may be used, e.g. RCC8, Egenhofer. These relations are generalized so that they may connect features as well as geometries.
 
-A Dimensionally Extended 9-Intersection Model (DE-9IM) pattern, which specifies the spatial dimension of the intersections of the interiors, boundaries and exteriors of two geometric objects, is used to describe each spatial relation. Possible pattern values are `-1` (empty), `0, 1, 2, T` (true) `= {0, 1, 2}, F` (false) = {-1}, *` (don’t care)` = {-1, 0, 1, 2}`. In the following descriptions, the notation `X/Y` is used denote applying a spatial relation to geometry types `X` and `Y` (i.e., `x` _relation_ `y` where `x` is of type `X` and `y` is of type `Y`). The symbol `P` is used for 0- dimensional geometries (e.g. points). The symbol `L` is used for 1-dimensional geometries (e.g. lines), and the symbol `A` is used for 2-dimensional geometries (e.g. polygons). Consult the Simple Features specification [ISO 19125-1] for a more detailed description of DE-9IM intersection patterns.
+A Dimensionally Extended 9-Intersection Model (DE-9IM) pattern, which specifies the spatial dimension of the intersections of the interiors, boundaries and exteriors of two geometric objects, is used to describe each spatial relation. Possible pattern values are `-1 (empty)`, `0, 1, 2, T (true) = {0, 1, 2}`, `F (false) = {-1}`, `* (don’t care) = {-1, 0, 1, 2}`. In the following descriptions, the notation `X/Y` is used denote applying a spatial relation to geometry types `X` and `Y` (i.e., `x` _relation_ `y` where `x` is of type `X` and `y` is of type `Y`). The symbol `P` is used for 0- dimensional geometries (e.g. points). The symbol `L` is used for 1-dimensional geometries (e.g. lines), and the symbol `A` is used for 2-dimensional geometries (e.g. polygons). Consult the Simple Features specification <<ISO19125-1>> for a more detailed description of DE-9IM intersection patterns.
 
-==== Parameters
+=== Parameters
 
 The following parameter is defined for the _Topology Vocabulary Extension_ requirements class.
 
 *relation_family*: Specifies the set of topological spatial relations to support.
 
-==== Requirements for Simple Features Relation Family (relation_family=Simple Features)
```

Comment:
I like the style of dropping "Requirements for" from the section titles. If this is what we want to do then we also need to drop "Requirements for" from titles in Sections 9, 10 and 11.

Reply:
Done in Commit https://github.com/opengeospatial/ogc-geosparql/pull/102/commits/72c2a7f59b7e4f1085f06553c4dfeafd0123976a